src/components/tabs/types.ts

```diff
@@ -91,4 +97,10 @@ export interface TabControlsProps {
   setActiveTabIndex: (newActiveIndex: number) => void
   ariaLabel?: TabsProps['ariaLabel']
   ariaLabelledBy?: TabsProps['ariaLabelledBy']
+
+  /**
+   * If true, nested tab styling will be applied to the component.
+   * Defaults to false.
+   */
+  isNested?: boolean
```

Comment:
Both types of tab controls now accept an `isNested` prop. When present, it applies nested styling to the tab controls. This prop is based on the tab nesting context, but _only if_ the consumer has also set `allowNestedStyles` on `Tabs`.